otx/algorithms/common/adapters/mmcv/hooks.py

```diff
@@ -491,8 +491,8 @@ def _init_rule(self, rule, key_indicator):
         self.key_indicator = key_indicator
         self.compare_func = self.rule_map[self.rule]
 
-    def _should_check_stopping(self, runner):
-        """Called _should_check_stopping in ReduceLROnPlateauLrUpdaterHook."""
+    def _is_check_timing(self, runner):
+        """Called _is_check_timing in ReduceLROnPlateauLrUpdaterHook."""
```

Comment:
This is meaningless. it's better to change it more meaningful.

Reply:
Reflected. Thank you.